pre_vs_berkeley.md

```diff
@@ -0,0 +1,440 @@
+|                                     | MassSpectrometryConfiguration              |                           |
+|                                     | MaterialProcessing                         |                           |
+|                                     | MobilePhaseSegment                         |                           |
+|                                     | NucleotideSequencing                       |                           |
+|                                     | PortionOfSubstance                         |                           |
+|                                     | ProtocolExecution                          |                           |
+|                                     | StorageProcess                             |                           |
+|                                     | WorkflowExecution                          |                           |
+
+For example, `MetagenomeAnnotationActivity` used to be located in the hierarchy
+`Activity.WorkFlowExecutionActivity`, but now it and 11 similar classes are located in
+`NamedThing.PlannedProcess.WorkflowExecution`. The word `Activity`, inherited from the Provenance Ontology, has largely
+been removed from the schema, as `PlannedProcess` from the Ontology of Biomedical Investigations now plays a more
+important organizational role.
+
+## v10.9.1
+
+![Activity.png](Activity.png)
+
+`ProtocolExecution` has been added as a `PlannedProcess` for aggregating other processes together when they follow the
+same `Protocol`.
+
+![ProtocolExecution.png](ProtocolExecution.png)
+
+Several other kinds of processual classes were also migrated into the `NamedThing.PlannedProcess`, with or without
+renaming or other refactoring. For example, the fairly vague root class `OmicsProcessing` is now
+`NamedThing.PlannedProcess.DataGeneration`, which now has the following subclasses:
+
+- `MassSpectrometry`
+- `NucleotideSequencing`
+
+The leagcy `BiosampleProcessing` class (including `Pooling`, `LibraryPreparation`) and some classes that were previously
+direct subclasses of `PlannedProcess` (`Extraction`, `SubSamplingProcess`, `MixingProcess`, `FiltrationProcess`,
+`ChromatographicSeparationProcess`) have been re-rooted into a new `MaterialProcessing` class.
+
+`DissolvingProcess` and `ChemicalConversionProcess` are new subclasses of `MaterialProcessing` as of v11.
+
+## v11.0.1
+
+![MaterialProcessing.png](MaterialProcessing.png)
+
+`StorageProcess` has been added as a new  `PlannedProcess` but is not considered `MaterialProcessing` because it does
+not intrinsically create new/modified samples.
+
+Terms that implied quantification like `MetaboliteQuantification` have been renamed to reemphasize the more fundamental
+activity of identifying, thus `MetaboliteIdentification`
+
+A `CalibrationInformation` class and two `Configuration` classes have been added in support of chromatographic
```

Comment:
> two `Configuration` classes

Is this referring to two classes, each of whose name was `Configuration`? Or is it referring to two classes that inherited from a class named `Configuration`? Or is it referring to classes whose names end with "...`Configuration`"? Etc.